### DISCUSSION TITLE
Verifying registry modules and supply chain issues

### DISCUSSION THREAD

**Discussion by jmreicha** [opentofu/opentofu#1540] [General]
I came across the following post and was wondering if the tofu registry has anything similar in place https://discuss.hashicorp.com/t/hcsec-2024-04-terraform-registry-module-supply-chain-security-improvements/62815.

If there isn’t anything in place, are there any plans for module trust verification and repo hijacking?

**Comment by ghost** [comment]
Hi @jmreicha I created an issue from your request (#1545) so the core team takes a look at it in the upcoming issue triages.
